topi/python/topi/x86/int8_intrinsics.py

```diff
@@ -0,0 +1,107 @@
+"""Core kernel of dot product of 4 Int8 operations"""
+#pylint: disable=invalid-name
+import tvm
```

Comment:
Let us rename it to tensor_intrin.py to be consistent with https://github.com/dmlc/tvm/pull/1707